### ISSUE TITLE
[bitnami/harbor] Wrong host string with sentinel enabled

### ISSUE THREAD

**Issue by lwj5** [bitnami/charts#11350]
### Name and Version

bitnami/harbor 15.0.0

### What steps will reproduce the bug?

1. enabled external redis sentinel
2. See redis host in all configmap as `redis.redis:26379/mymaster:6379`

### What is the expected behavior?

2. See redis host in all configmap as `redis.redis:26379/mymaster`

### What do you see instead?

Cannot connect to redis

**Comment by RichardSufliarsky** [comment]
Try to use
```
externalRedis:
  host: ""
```
in your values.yaml.
Check also https://github.com/bitnami/charts/issues/11266 and https://github.com/bitnami/charts/issues/11354.

**Comment by lwj5** [comment]
@RichardSufliarsky it would work with host: ""

if you look at the configmaps `{{ printf "%s:%s" ( include "harbor.redis.host" . ) ( include "harbor.redis.port" . ) | quote }}`
The 2nd colon will always be there anyways.

**Comment by RichardSufliarsky** [comment]
> @RichardSufliarsky do you mean port: ""? it wouldn't work
> 
> if you look at the configmaps `{{ printf "%s:%s" ( include "harbor.redis.host" . ) ( include "harbor.redis.port" . ) | quote }}` The 2nd colon will always be there anyways.
> 
> Unfortunately, I do not have time to fix this chart, so I just edited the values of the affected config maps

No, I mean host: "" in externalRedis, but you are right, it won't help. I looked at goharbor's helm chart and there seems to be different config for Sentinel: https://github.com/goharbor/harbor-helm/blob/9c62943736458b63cb8d8f1bd347e8cdfb34a1f9/templates/registry/registry-cm.yaml#L173
And they handle addr correctly: https://github.com/goharbor/harbor-helm/blob/9c62943736458b63cb8d8f1bd347e8cdfb34a1f9/templates/_helpers.tpl#L155

**Comment by lwj5** [comment]
not stale
